Dockerfile

```diff
@@ -454,12 +455,16 @@ FROM alpine:3.11 AS installer
 RUN apk add --no-cache --update \
     bash \
     ca-certificates \
-    cdrkit \
     efibootmgr \
+    grub \
```

Comment:
I need to figure out how to compile a font that is required by `grub-mkrescue`. Then we can remove this.